src/messages/services/terms/index.js

```diff
@@ -3,16 +3,21 @@ import { ZalgoPromise } from 'zalgo-promise/src';
 
 import { memoizeOnProps } from '../../../utils';
 
+const currenyMap = {
+    US: 'USD',
+    DE: 'EUR'
+};
+
 function assembleUrl({ account, amount }) {
     const baseUrl = __MESSAGES__.__TERMS_URL__;
     const queries = [
         'json=true',
+        // 'mid=8SUQZGVVX324C'
         stringStartsWith(account, 'client-id') ? `cid=${account.slice(10)}` : `mid=${account}`
     ];
 
-    // TODO: Look to dynamically set these values as we push for location driven logic
-    queries.push('country=US');
-    queries.push('currency=USD');
+    queries.push(`country=${__MESSAGES__.__LOCALE__}`);
+    queries.push(`currency=${currenyMap[__MESSAGES__.__LOCALE__]}`);
```

Comment:
This is similar to how it was previously, but can we just add these directly when creating `queries`?